src/content/admin/registry-credentials/amazon-ecr.mdx

```diff
@@ -55,7 +59,83 @@ Remember the value of `AccessKeyId` and `SecretAccessKey`. You will need them in
 
 Add the following registry credentials to the [Admin Registry Credentials view](index.mdx#add-registry-credentials):
 
-- **Type**: `AWS`
+- **Type**: `AWS IAM User`
 - **Hostname**: the default registry endpoint is `https://{AWS_ACCOUNT_ID}.dkr.ecr.{REGION}.amazonaws.com`
 - **Username**: `AccessKeyId` from the previous step
 - **Password**: `SecretAccessKey` from the previous step
+
+# Using OIDC Federation
+
+## Step 1: Create the Identity Provider
+
+```bash
+OIDC_ENDPOINT=https://container.googleapis.com/v1/projects/my-project-12345/locations/us-central1/clusters/development
```

Comment:
are we showing this endpoint in the Okteto UI? If yes, let's document how to get it from there

Reply:
I think frontend is done already and available in staging